lib/vmpooler/api/helpers.rb

```diff
@@ -54,35 +54,63 @@ def authorized?
         return false
       end
 
+      def authenticate_ldap(port, host, user_object, base, username_str, password_str)
+        require 'rubygems'
+        require 'net/ldap'
```

Comment:
@mattkirby Just curious why you are putting the require statement inside the authenticate_ldap method instead of at the top of the file.  Is there a reason to limit the scope?

Reply:
Only because it's what was already being done. It probably should be at the top of the file.